### ISSUE TITLE
[BUG/ISSUE] Segmentation fault when running a nested simulation of AS using GCClassic 13.1.0

### ISSUE THREAD

**Issue by Alex-Fang66** [geoschem/geos-chem#786]
Hi, again.
### Description of the problem
I was trying to run a 05x0625 nested simulation of AS. However, the SGE jobs stopped with a error message of Segmentation fault every time when the GC ran to  'C3H8 (via UCX rates)'. And there was not any clear bug in HEMCO.log and GC.log.

### Description of troubleshooting performed
I saw others encountered the segmentation fault due to ignorance of setting OPM_STACKSIZE of Memory, but I have already set them in my sub.sh which is attached in run_log. Maybe it was the setting in my HEMCO_Config.rc that did not suit for AS or something...

### GEOS-Chem version
GCClassic 13.1.0

### Log files
- Build log (if applicable): attached in run_log
- Run logs (if applicable): attached in run_log
- Run script (if applicable): attached in run_log
[run_log.zip](https://github.com/geoschem/geos-chem/files/6860202/run_log.zip)

### Software versions
- CMake version: cmake@3.16.1
- Compilers (Intel or GNU, and version): GNU gcc@9.3.0
- NetCDF version:netcdf-fortran@4.5.3

**Comment by lizziel** [comment]
Hi @Alex-Fang66, have you tried rebuilding with debug flags? Rerunning after doing this may give more information on what the issue is. To build with debug mode on append `-DCMAKE_BUILD_TYPE=Debug` to the cmake command.

**Comment by Alex-Fang66** [comment]
@lizziel  Thanks for replying. I tried rebuilding with debug flags and got some error information, but I cannot understand it at all. I will attach the report below, and hope to hear from you soon.
[sub.txt](https://github.com/geoschem/geos-chem/files/6866762/sub.txt)

**Comment by yantosca** [comment]
Thanks for the feedback @Alex-Fang66.  If you look in the traceback you will find:
```
#2  0x8ffb6a in __calc_met_mod_MOD_airqnt._omp_fn.0
	at /home/fangli/gc_merra2_fullchem/CodeDir/src/GEOS-Chem/GeosCore/calc_met_mod.F90:296
```
so this is telling you an error occurred at line 296 of `GeosCore/calc_met_mod.F90`, which is the middle of this parallel loop:

https://github.com/geoschem/geos-chem/blob/d7baeb8fe51a66914408e9d1f655dd062bd5d4df/GeosCore/calc_met_mod.F90#L275-L311

If you have rebuilt the code with debug flags, and you have a core file, you can use the gdb debugger to find out more about where the code stopped:
```
gdb gcclassic core.XXXXX
```
where XXXXX is the number in the core file name.  This will take you to the point in the code where the error occurred.  You can then use some basic commands like
```
print I
print J
print State_Met%LWI(I,J)
```
etc. to explore the values of variables.  That will help you to narrow down if you have a division by zero or other floating-point error going on.

**Comment by Alex-Fang66** [comment]
@yantosca Thank you for your guidance.
I tried using gdb to debug, but didn't print out anything. Here is the message from command 'bt'.
![gdb](https://user-images.githubusercontent.com/65645232/126859638-3aaa8bca-ddb1-4e5d-b968-bc82f4c2be4d.png)
[gdb.txt](https://github.com/geoschem/geos-chem/files/6871992/gdb.txt)
I don't konw if this SIGFPE error comes from AFCID which I closed due to another error. So I will put the error message with AFCID on.
[AFCID.txt](https://github.com/geoschem/geos-chem/files/6871998/AFCID.txt)
I also tried close the MEIC, and the bug is same as its on. I am not sure if it is okay to only make MEIC on in regional inventory ...  
It's really confusing why this happened since 2x25 simulation of global works well.

**Comment by Alex-Fang66** [comment]
@yantosca Hi, Could you tell me what to do next?  I really have no idea how to deal with it right now.

**Comment by yantosca** [comment]
@Alex-Fang66, here's what you should try:

  - Start with a fresh "out-of-the-box" version of GEOS-Chem, see if that compiles and runs.
  - Add one change at a time (i.e. turn on MEIC) and then re-run
  - Repeat until you find the change that caused the issue

**Comment by Alex-Fang66** [comment]
@yantosca Thank you, I will try soon!

**Comment by yantosca** [comment]
This appears to be a duplicate of #788

**Comment by yantosca** [comment]
The fix that I described in #798 allowed me to run a nested-grid AS simulation to completion.  Therefore I will close out this issue.  Feel free to reach out again if you are still having problems.
